### ISSUE TITLE
Appveyor errors when persisting temp files

### ISSUE THREAD

**Issue by heartsucker** [heartsucker/rust-tuf#131]
Possibly related to permissions?

https://ci.appveyor.com/project/heartsucker/rust-tuf/build/302/job/rg32sb094w5oxydn

```
---- repository::test::file_system_repo_targets stdout ----
	thread 'repository::test::file_system_repo_targets' panicked at 'called `Result::unwrap()` on an `Err` value: Opaque("Error persisting temp file: PersistError { error: Error { repr: Os { code: 5, message: \"Access is denied.\" } }, file: NamedTempFile(\"C:\\\\Users\\\\appveyor\\\\AppData\\\\Local\\\\Temp\\\\1\\\\rust-tuf.V4hIuJmhxxk0\\\\temp\\\\.tmpoqHfon\") }")', src\libcore\result.rs:860:4
stack backtrace:
   0:   0xb6b03c - std::panicking::default_hook::{{closure}}::h48c07d1bc07c0089
   1:   0xb6ad55 - std::panicking::default_hook::h72dcd99c4287a2b2
   2:   0xb6b3dc - std::panicking::rust_panic_with_hook::h5b41b898e5797800
   3:   0xb6b2fe - std::panicking::begin_panic_new::hcf5fda792d4e8857
   4:   0xb6b242 - std::panicking::begin_panic_fmt::h673341051b2e87ff
   5:   0xb6b1c9 - rust_begin_unwind
   6:   0xb7df4a - core::panicking::panic_fmt::hfda8af447f9f88e7
   7:   0x4f924c - core::result::unwrap_failed::hd388f4b0cb245d3c
                       at C:\projects\rust\src\libcore/macros.rs:41
   8:   0x4c6580 - <core::result::Result<T, E>>::unwrap::hfe9570c1481489b0
                       at C:\projects\rust\src\libcore/result.rs:738
   9:   0x60eafd - tuf::repository::test::file_system_repo_targets::h026b0e12dfcc62e6
                       at src/repository.rs:557
  10:   0x9f540e - <F as test::FnBox<T>>::call_box::h05ad4323db8f2f84
  11:   0x9e862d - std::panicking::try::do_call::h894b4439d97b361e
  12:   0xb6e42a - __rust_maybe_catch_panic
  13:   0x9e7b00 - std::sys_common::backtrace::__rust_begin_short_backtrace::hf05802d444e79d0c
  14:   0x9e860b - std::panicking::try::do_call::h7013ce95188dd650
  15:   0xb6e42a - __rust_maybe_catch_panic
  16:   0x9eeea0 - <F as alloc::boxed::FnBox<A>>::call_box::hc3e3a2aefef83630
  17:   0xb69810 - _ZN3std3sys3imp6thread6Thread3new12thread_start17h77c42d6cad152acaE@4
  18: 0x75477c03 - x25519_scalar_mult_generic
  19: 0x7782ad2e - x25519_scalar_mult_generic
```

**Comment by heartsucker** [comment]
Apparently this error has existed for a month, and appveyor stopped sending me emails about failures. Sorry, everyone.

**Comment by heartsucker** [comment]
Closed by #135
